curriculum/challenges/english/08-coding-interview-prep/rosetta-code/deal-cards-for-freecell.english.md

```diff
@@ -76,11 +76,11 @@ Deals can also be checked against <a href="https://freecellgamesolutions.com/" t
 ```yml
 tests:
   - text: <code>dealFreeCell</code> is a function.
-    testString: assert(typeof dealFreeCell === 'function', '<code>dealFreeCell</code> is a function.');
+    testString: assert(typeof dealFreeCell === 'function');
   - text: <code>dealFreeCell(seed)</code> should return an object.
-    testString: assert(typeof dealFreeCell(1) === 'object', '<code>dealFreeCell(seed)</code> should return an object.');
+    testString: assert(typeof dealFreeCell(1) === 'object');
   - text: <code>dealFreeCell(seed)</code> should return an array of length 7.
-    testString: assert(dealFreeCell(1).length === 7, '<code>dealFreeCell(seed)</code> should return an array of length 7.');
+    testString: assert(dealFreeCell(1).length === 7);
   - text: "<code>dealFreeCell(1)</code> should return an array identical to example \"Game #1\""
     testString: "assert.deepEqual(dealFreeCell(1), game1, '<code>dealFreeCell(1)</code> should return an array identical to example \"Game #1\"');"
```

Comment:
2 message arguments need to be removed in this file.

Reply:
Fixed in separate commit.